src/lib/p5-components/components/Vector.svelte

```diff
@@ -21,31 +21,31 @@
   const CONE_HEIGHT = 0.5;
   const CONE_DIAMETER = 0.1;
 
-  let draggables = getCanvasContext().draggables;
+  // let draggables = getCanvasContext().draggables;
   const key = Symbol('vector');
 
   // NOTE: draggables won't get updated if the Vector is moved by some other means, e.g. sliders (because I don't see an elegant way of doing that without causing a cyclical dependency in reactivity)
-  $: {
-    if (draggable) {
-      let draggablePos =
-        $draggables.get(key) || origin.clone().add(direction.clone().multiplyScalar(length)); // TODO: fix undefined thingy here
-      let v = draggablePos.clone().sub(origin);
-      direction = v.clone().normalize();
-      length = v.length();
-    }
-  }
+  // $: {
+  //   if (draggable && false) {
+  //     let draggablePos =
+  //       $draggables.get(key) || origin.clone().add(direction.clone().multiplyScalar(length)); // TODO: fix undefined thingy here
+  //     let v = draggablePos.clone().sub(origin);
+  //     direction = v.clone().normalize();
+  //     length = v.length();
+  //   }
+  // }
```

Comment:
Some commented out stuff here from my old implementation that I think can be removed

Reply:
Is removed in the latest version